### PR TITLE
Add Redux into Summary page

### PR DESCRIPTION
**Pull request by itssumertime**
Removes hardcoded data and now displays data that a user entered.

### CODE REVIEW

File: client/src/components/SummaryPage.jsx [hunk 1]
```diff
@@ -1,25 +1,18 @@
...
-/**
- * PLACEHOLDER data being used until the redux store is connected.
- */
-const data_ = {
-  HOUSEHOLD_SIZE: 5,
-  IS_EMPLOYED: true,
-};
-
-const ResponsesContext = React.createContext(data_);
+import React, { useContext } from "react";
```
Comment (egracer) [inline client/src/components/SummaryPage.jsx:1]: You can remove `useContext`

Comment (mscharfstein) [reply client/src/components/SummaryPage.jsx:1]: and run yarn lint --fix :(

Comment (mscharfstein) [reply client/src/components/SummaryPage.jsx:1]: and run yarn lint --fix :(

Comment (itssumertime) [reply client/src/components/SummaryPage.jsx]: Done, thanks!

**Pull request merged**